BugsnagRequestMonitor/BugsnagRequestMonitor/NSURLSession+Tracing.m

```diff
@@ -0,0 +1,67 @@
+//
+//  NSURLSession+Tracing.m
+//  
+//
+//  Created by Karl Stenerud on 07.09.21.
+//
+
+#import "NSURLSession+Tracing.h"
+#import "BSGURLSessionTracingDelegate.h"
+#import "BSGURLSessionTracingProxy.h"
+#import <objc/runtime.h>
+
+
+static IMP get_class_imp(Class _Nonnull class, SEL selector) {
+    Method method = class_getClassMethod(class, selector);
+    return method_getImplementation(method);
+}
```

Comment:
This function is unnecessary because `method_setImplementation` returns the previous implementation.